docs/README.rst

```diff
@@ -62,20 +51,14 @@ from GitHub by running the following command:
 
     source .venv/bin/activate # activate the `venv` virtual environment
 
-3.3. Install Dependencies
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-After cloning the ``schematic`` project from GitHub and setting up your
-virtual environment:
+3.2. Installing
+~~~~~~~~~~~~~~~~~
 
 .. code:: bash
 
-    cd schematic  # change directory to schematic
-    git checkout develop  # switch to develop branch of schematic 
-    poetry build # build source and wheel archives
-    pip install dist/schematicpy-0.1.11-py3-none-any.whl  # install wheel file
+    pip install -U schematicpy  # install and upgrade package
```

Comment:
See my comment on using `python -m pip ...`. [Source](https://snarky.ca/why-you-should-use-python-m-pip/).

Reply:
This wasn't fixed? I don't think you need the `-U` if you use `python -m pip install ...`.

Reply:
Ah my bad, I missed updating this in the `docs/README.rst` file.